code/elsdsr/data_item.py

```diff
@@ -0,0 +1,80 @@
+
+import os.path as path
+import textwrap
+
+import numpy as np
+import scipy
+import scipy.io.wavfile
+import scipy.signal
+
+thisdir = path.dirname(path.realpath(__file__))
+elsdsrdir = path.realpath(path.join(thisdir, "..", "..", "data", "elsdsr"))
+
+class DataItem:
+    def __init__(self, usage, sex, speaker, filename):
+        self.file = filename[len(elsdsrdir):]
+        self.usage = usage
+        self.sex = sex
+        self.speaker = speaker
+
+        self.wav = filename
+
+        self._signal_shape = None
+        self.rate = 16000  # This is the same for all files in timit
```

Comment:
Ja, det kører fint for `sex_classifier.py` med en fejl ca. 0.03, hvis man truncater til 300kolonner, så ca. det samme som det andet.